PERFORMANCE_BENCHMARKS.md

```diff
@@ -0,0 +1,112 @@
+# README: Running Performance Benchmarks on Pull Requests
+
+## Overview
+
+`benchmark-pull-request` GitHub Actions workflow is designed to automatically run performance benchmarks on a pull request when a specific comment is made on the related issue. This ensures that performance benchmarks are consistently and accurately applied to code changes, helping maintain the performance standards of the repository.
```

Comment:
I would expect the comment to be on the pull request, where the code lives. Plus, the issue could have several pull requests "associated" with it, fe multiple backports or large features split in chunks.

Reply:
Sorry for the confusion, the comment has to be added on the pull request and not on any issue. Will correct the wording.

Reply:
Could we please updated docs accordingly?

Reply:
Done!